service/storage/storage.go

```diff
@@ -366,9 +362,12 @@ func (s *service) SyncFile(ctx context.Context, bucketID, fileID, version string
 		return fd, ErrAlreadyExists
 	// Already exists and conflicts
 	case err == nil && checksum != fd.Checksum:
-		s.logger.Error().
-			Msg("File already exists and has conflicting checksum")
-		return nil, ErrAlreadyExistsConflict
+		s.logger.Debug().
```

Comment:
Can we bump this message to `Info` so we can observe it on staging and production without all the debug noise?